webpack/webpack.config.dev-server.js

```diff
@@ -1,38 +1,43 @@
 var path = require('path');
 var webpack = require('webpack');
-var commonLoaders = require('./common.config').commonLoaders;
-var assetsPath = require('./common.config').assetsPath;
-var publicPath = require('./common.config').publicPath;
+
+var commonConfig = require('./common.config');
+
+var commonLoaders = commonConfig.commonLoaders;
+var publicPath = commonConfig.output.publicPath;
+var externals = commonConfig.externals;
 
 module.exports = {
```

Comment:
My hunch is that postcss-import is a culprit - we might need to bump the version up :)

Stay tuned - will be done tomorrow morning (if i wake up haha)